packages/Popover/src/Popover.story.js

```diff
@@ -68,4 +68,41 @@ function Example() {
   );
 }
 
-storiesOf('Popover', module).add('Default', () => <Example />);
+function AdvancedExample() {
+  const [active, setActive] = useState(false);
+  const refEl = useRef(null);
+
+  return (
+    <button
+      ref={refEl}
+      onClick={() => setActive(!active)}
+      className={cx(
+        containerStyle,
+        referenceElPositions[
+          select(
+            'Reference Element Position',
+            ['center', 'top', 'right', 'bottom', 'left'],
+            'center',
+          )
+        ],
+      )}
+    >
+      {text('Button Content', 'Popover')}
+      <Popover
```

Comment:
Let's show Popover doesn't have to be a child of the reference element, to really drive home the flexibility of the advanced usage.